### PR TITLE
Seelie(Pixie)-PR

### PR DESCRIPTION
**Pull request by Simply-Sydney**
## About The Pull Request

Strap in folks, its a long one.

Seelies have come to Rockhill! What were formerly referred to as Pixies have become Seelies, the first mechanically unique race to (hopefully) be added to the game.

image
^ Seelie Lore

Seelie are weak creatures, and shouldnt ever see combat unless absolutely necessary. To reflect this, they have 1 strength and firing a bow will inflict only 50% damage

SeelieStats
^ Seelie Stats

Seelie also have their own custom role! This race specific job is meant to be a supporting one, similar to beggars, in which the Seelie will use their spells, abilities, and RP to open up fun new interactions for what otherwise would be boring jobs. Due to Seelie stats, players will have an extremely bad time trying to swing a hammer, pickaxe, or axe to do a typical job. This is on purpose to discourage players from deviating from the primary 'support' intention

TownSeelieMessage
^ Town Seelie Job, limited in both PQ requirement and number (6 only)

Of course this also means that Seelie must be disallowed from other roles that simply don't make sense:

AvailableJobs
LatejoinRoles
LatejoinRoles2
^ Jobs Available (NOTE: Prisoner Seelie will NOT get access to Seelie spells!)

The Seelie themselves have backgrounds / races called "Elemental Attunments", which in lore is the specific element nearby when a Seelie first came into existance (the do not have families or reproduce after all)

image
ElementalConnection
^ Race Choice

In addition to skin coloration, the Seelie also have the ability to select their own custom wing color

WingColorCustomizer
^ Wing Feature

TownSeelieLoadout
^ A Seelie equiped under the Town Seelie loadout, cute little thing isn't she?

To support their role, and for flavor, Seelie have access to a few custom and unique spells!

image
^ Spell names and icons

These spells are not hugely impactful, have long cooldowns, and take a fair amount more fatigue to cast than others seen in the game. They are meant to add flavor and enhance the round in quirky new ways. (Call Beast also has a 1/100 chance to spawn a Rous instead of a Rat)

Taking a closer look at these spells, you can see how they reflect in game:
Animate Object works on anything that is not a structure or machine. As such it will not cause door movement, wall movement, or anything that a blacksmith would like to stay in place for instance
AnimateObject2
^An animated plant! (Note they do turn hostile but are unable to attack at all)
AnimateObject4
^Example of it not working on structures

Animate Object is also very, very exhausting for the Seelie, the toughest and most flavorful spell by far!
AnimateObject3

To replenish this exhaustion a little, the Seelie also has access to a "Drain" spell. While it does look similar to something a vampire may have, I can assure you its not and only drains fatigue (blue bar) for casting spells. It is also a very long cooldown and doesnt drain tooooo much~
Drain2
^ Caster text
Drain3
^ Target text

The next spell, Call Beast, works to spawn a rat nearby to the Seelie. They have a connection to nature after all, and this includes critters. This may be useful for hunters as well, as when one rat eats another.... well, youll have to see for yourself!

SummonBeast
^ Call Beast spell

Speaking of that connection to nature, a commonly requested feature has been a loot bush replenishment. Well, Seelie can do that for you! As with all the other spells, cooldown can be quite long but not as much so as the others

ReplenishNature2
^ Casting replenish nature on an EMPTY bush will refill it a little

A much more mischievous spell, but also usefully for any looking to heal a wounded chest fast, Strip Shirt will force remove any NON armor clothing. I say again, this will NOT affect armor and should not either:

StripShirt_notArmor
^ Stripping the merchants top off in style

The final spell is to make the target trip balls. Called Seelie Dust, it will afflict the target with a custom drug only attainable from Seelies, this will change the targets stats and hopefully make their job a little easier and more fun! Note that it is possible to overdose, but the limit for doing so is very high and the cooldown for Seelie Dust makes this very difficult to reach

SeelieDust2
SeelieDust_Slurring
SeelieDust3
^ It also causes slurring and drooling

While not a spell that can be selected and casted by the Seelie player, Seelie also have an 'Aura' which will affect anyone around them long enough. Whether this aura is beneficial to those around them or harmful is changed by whether the Seelie is in a good mood or bad. Bad mood pictured below, but good mood does the opposite for anyone adjacent to the Seelie:

SeelieCurse
SeelieCurse2

Right, lets change gears and get to the mechanics. Seelie feel and play very differently to any other race. They can passover tables, occupy the same space as other people, take no fall damage or tile slowdown, and even hover straight over water tiles! (WHILE STANDING ONLY), they also fall down from exhaustion right out of the air!:

Passthrough1
FallDamage-zLevel2
FallDamage2
WaterHover1
WaterHover2
FallFromExhaustion1
Seelie also give off a very small amount of natural light:
NaturalLight

Other interactions still have been disabled, such as carrying people, for both realism and to balance the fact that these things are quick and have no mud tile slowdown. They will also slowdown from non-upgraded human grabs, and cannot grab structures such as chests and handcarts at all:

GrabbingCarrying1
Pillory1

Of course this also includes ERP interactions, as shown in the menu below many unbelievable and fetish territory ERP verbs have been disabled. And while not shown here explicitly, the flavor text for the remaining ones have been altered if its done from a Seelie -> Humen or vice versa

FromSeelieToHumen
FromHumenToSeelie
A warning, some interactions such as penetrative sex will harm the Seelie to a massive extent. While not killing them outright, the Seelie will wish they were dead:
Sex

The final thing to cover would be what other races can do to Seelie. Weve seen how Seelie players interact with the world, but other races can do unique things to them as well!
For example, Seelie fit inside lanterns! To open and close the door to the lantern, one must right click it in their hand.
There are checks in place to make sure that the lantern is not already lit, as it would burn the Seelie, and also ensure that a lantern with an occupant cannot be lit. These give off a nice bluish light when occupied by a Seelie:

NOTE: To load a Seelie, you must be on WEAK intent
Lantern1
Lantern2
Lantern3
Lantern4
A Seelie locked in a lantern against its will can always try to bust out of it with the resist key. A lantern with the door open will let the Seelie out upon the Seelie player moving:
Lantern6
Lantern7

One final thing remains. A Seelie's wings are very valuable. An unscrupulous fellow with no morals or regard for others may take advantage of this and rip the wings straight from a Seelie's back! To do this, he must first GRAB the Seelie, then MMB it while its grabbed:
WingRip1
Keen observers may note that a curse is placed upon the poor fool who ripped the wings, this can be prevented only one way (a gagged mouth, not grab) and will inflict temporary deafness and confusion on the person to do this
WingRip2
^ This is what happened to the Seelie
WingRip3
WingRip4

This will of course kill the Seelie, almost immediately their heart stops and spine breaks gruesomely. Miracles will not even keep them alive. 
This is intentional to prevent seeing wingless Seelie running about, and some animations / pixel offsets may be problematic for a living moving wingless Seelie

Final note: Due to the problem of Seelie without wings appearing childlike in nature, reviving one that has had its wings pulled will always cause them to grow back. There should NEVER be a living wingless Seelie

## Why It's Good For The Game

Whew you made it to here, thanks for reading everything I know its a lot of stuff.

Personally, I think this is good for the game as it opens up brand new roleplay opportunities and brings some fun to towner jobs that are otherwise boring by keeping Seelie around as work companions. The Seelie can bless them with their aura for extra luck, make them high on Seelie dust, or be a chaotic menace as shown through the spells and mechanics.

Being mechanically unique and not simply another reskin/statchange race hopefully will make for a new precedent in races design going forward as well. While Seelie seem cute, some of the mechanics implemented are obviously very dark in nature, and I believe that is fitting for the game more-so than most other parts of the game.

MASSIVE CREDIT AND SHOUTOUT TO THEFENIX323
~ Couldn't have done it without them ~
And to all others who helped make this as great as it is, you are all truly wonderful!

### CODE REVIEW

**Comment by Simply-Sydney** [comment]
Quick editors note here: the font for the Faexin language they speak is almost completely unreadable. Its a file that was already in game when I started and Id like to replace it someday if it becomes an issue:
Faexin1
Faexin2

**Comment by CalamaBanana** [comment]
>While not killing them outright, the Seelie will wish they were dead

If not already in, any interaction that would cause serious harm needs a warning prompt, similar to that prompt one gets when trying to force it on defiant players. People need it shoved in their face that it causes deadly harm, else they press the button out of curiosity and accidentally murder pixies.

>A much more mischievous spell, but also usefully for any looking to heal a wounded chest fast, Strip Shirt will force remove any NON armor clothing.

I highly recommend to not include this spell in that form. Instantly stripping a woman's chest naked in the streets is not a mischievous little prank, but somewhere between molestation and violation. A lot of funny pixies will use it to completely ruin the serious mood of ongoing scenes, and then ahelp after they get the consequences for it. If you want it to be a prank, have it aim at shoes or hats randomly, not chest. That would be innocent fun.


Other than that, as someone who liked RPing pixies in other settings, I am really curious to see how they play out here. I appreciate how much effort you put into this.

**Comment by Simply-Sydney** [comment]
> > While not killing them outright, the Seelie will wish they were dead
> 
> If not already in, any interaction that would cause serious harm needs a warning prompt, similar to that prompt one gets when trying to force it on defiant players. People need it shoved in their face that it causes deadly harm, else they press the button out of curiosity and accidentally murder pixies.
> 
> > A much more mischievous spell, but also usefully for any looking to heal a wounded chest fast, Strip Shirt will force remove any NON armor clothing.
> 
> I highly recommend to not include this spell in that form. Instantly stripping a woman's chest naked in the streets is not a mischievous little prank, but somewhere between molestation and violation. A lot of funny pixies will use it to completely ruin the serious mood of ongoing scenes, and then ahelp after they get the consequences for it. If you want it to be a prank, have it aim at shoes or hats randomly, not chest. That would be innocent fun.
> 
> Other than that, as someone who liked RPing pixies in other settings, I am really curious to see how they play out here. I appreciate how much effort you put into this.

Yeah I probably will alter the spell to target feet-hat-gloves, youre right the shirt is a little overly egregious.

As for the fucking, I am massively against that. It will not kill the Seelie, and I believe that things like "massive insertion into tiny thing" should be either inherently obvious or a learned thing for those in game who have no common sense. 
A popup to prevent people from doing dumb shit would entirely remove the immersion and atmosphere of whats supposed to be a dark setting.

**Comment by pilotmans** [comment]
oh shit it's the fae

**Comment by kepesk1** [comment]
I like it

**Comment by Simply-Sydney** [comment]
Updated the stripping spell to randomly target boots-gloves-head, will NOT strip helms or head slot armor for balance reasons.
Also updated standing / laying to disallow for any wingless Seelie to stand altogether, in the case of surgical wing removal circumventing death

image

**Comment by Coolgat3** [comment]
Idk about letting them use bows even at 50% damage tbh

**Comment by Coolgat3** [comment]
> I can't say how well/poorly works in RT's combat system, but one thing that's an issue in SS13 games as a whole is that _very_ small sprites, like say a pixie, are incredibly difficult to _click_ in comparison to a person. This has issues when you need to click on someone to, say, attack them or grab them, but also for things if you want to examine or so on.
> 
> If you haven't already, you may want to give pixie a translucent backdrop to their sprite so they still have a relatively large click area without it being visible. This has it's own issues if someone is someone is trying to click _around_ the seelie, but no more then a larger person.
> 
> Also; a seelie should be able to ride on someone's shoulders/head, because it's cute.

you don't need to click the sprite to attack someone the combat is attack on tile based

**Comment by DorfGuy1** [comment]
> I can't say how well/poorly works in RT's combat system, but one thing that's an issue in SS13 games as a whole is that _very_ small sprites, like say a pixie, are incredibly difficult to _click_ in comparison to a person. This has issues when you need to click on someone to, say, attack them or grab them, but also for things if you want to examine or so on.
> 
> If you haven't already, you may want to give pixie a translucent backdrop to their sprite so they still have a relatively large click area without it being visible. This has it's own issues if someone is someone is trying to click _around_ the seelie, but no more then a larger person.
> 
> Also; a seelie should be able to ride on someone's shoulders/head, because it's cute.

I agree with the shoulder riding statement. Also making the little buggers be hard to click on will be thematic I feel.

**Comment by EzekyalLegit** [comment]
Is this seriously a female only race? Disappointing.

**Comment by Simply-Sydney** [comment]
> Is this seriously a female only race? Disappointing.

They're meant to be an antithesis to goblins, which are an all male race.
Its something easily changed if its asked for enough

**Comment by pilotmans** [comment]
> > Is this seriously a female only race? Disappointing.
> 
> 
> 
> They're meant to be an antithesis to goblins, which are an all male race.
> 
> Its something easily changed if its asked for enough

I don't see much of a reason to change it otherwise though.

**Comment by EzekyalLegit** [comment]
> > Is this seriously a female only race? Disappointing.
> 
> They're meant to be an antithesis to goblins, which are an all male race. Its something easily changed if its asked for enough

I would much rather be a Seelie than a filthy graggarite goblin but it's your vision.

**Comment by EzekyalLegit** [comment]
> > > Is this seriously a female only race? Disappointing.
> > 
> > 
> > They're meant to be an antithesis to goblins, which are an all male race.
> > Its something easily changed if its asked for enough
> 
> I don't see much of a reason to change it otherwise though.

So people who like faeries and roleplay males predominantly can have male Seelie characters.

**Comment by pilotmans** [comment]
> > > > Is this seriously a female only race? Disappointing.
> 
> > > 
> 
> > > 
> 
> > > They're meant to be an antithesis to goblins, which are an all male race.
> 
> > > Its something easily changed if its asked for enough
> 
> > 
> 
> > I don't see much of a reason to change it otherwise though.
> 
> 
> 
> So people who like faeries and roleplay males predominantly can have male Seelie characters.

And yet anybody capable of playing this specific species to a high degree of effectiveness would also be able to roleplay as both sexes effectively, that and the fact that making seelies sexless probably wouldn't actually change anything to a significant degree because sexual features and mannerisms play such a little part in how fae act or at least what I assume how a fae should act.

Either way it seems one of the species main gimmicks is sex = death so take that as you will into consideration.

**Comment by EzekyalLegit** [comment]
> > > > > Is this seriously a female only race? Disappointing.
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > They're meant to be an antithesis to goblins, which are an all male race.
> > 
> > 
> > > > Its something easily changed if its asked for enough
> > 
> > 
> > > 
> > 
> > 
> > > I don't see much of a reason to change it otherwise though.
> > 
> > 
> > So people who like faeries and roleplay males predominantly can have male Seelie characters.
> 
> And yet anybody capable of playing this specific species to a high degree of effectiveness would also be able to roleplay as both sexes effectively, that and the fact that making seelies sexless probably wouldn't actually change anything to a significant degree because sexual features and mannerisms play such a little part in how fae act or at least what I assume how a fae should act.
> 
> Either way it seems one of the species main gimmicks is sex = death so take that as you will into consideration.

I would play gender neutral they/them Seelie for sure.

**Comment by Simply-Sydney** [comment]
> > > > > > Is this seriously a female only race? Disappointing.
> > > 
> > > 
> > > > >
> > > 
> > > 
> > > > >
> > > 
> > > 
> > > > > They're meant to be an antithesis to goblins, which are an all male race.
> > > 
> > > 
> > > > > Its something easily changed if its asked for enough
> > > 
> > > 
> > > > 
> > > 
> > > 
> > > > I don't see much of a reason to change it otherwise though.
> > > 
> > > 
> > > So people who like faeries and roleplay males predominantly can have male Seelie characters.
> > 
> > 
> > And yet anybody capable of playing this specific species to a high degree of effectiveness would also be able to roleplay as both sexes effectively, that and the fact that making seelies sexless probably wouldn't actually change anything to a significant degree because sexual features and mannerisms play such a little part in how fae act or at least what I assume how a fae should act.
> > Either way it seems one of the species main gimmicks is sex = death so take that as you will into consideration.
> 
> I would play gender neutral they/them Seelie for sure.

That was the original intent, since they don't breed and propagate in a different a-sexual way. However, that turned out to be WAY harder to code unfortunately. Everything is changeable in the future though!

**Comment by EzekyalLegit** [comment]
Be real though goblins will not be male only, busty green shortstacks are a very very common brand.

**Comment by pilotmans** [comment]
> Be real though goblins will not be male only, busty green shortstacks are a very very common brand.

Note: skeletons are all male with functional genitals on them, the same cannot be said for goblins which is a tad confusing.

**Comment by EzekyalLegit** [comment]
> > Be real though goblins will not be male only, busty green shortstacks are a very very common brand.
> 
> Note: skeletons are all male with functional genitals on them, the same cannot be said for goblins which is a tad confusing.

This change is how most players who want to be a female with a cock get their cock.

**Comment by pilotmans** [comment]
> > > Be real though goblins will not be male only, busty green shortstacks are a very very common brand.
> 
> > 
> 
> > Note: skeletons are all male with functional genitals on them, the same cannot be said for goblins which is a tad confusing.
> 
> 
> 
> This change is how most players who want to be a female with a cock get their cock. Harvesting from skelelons.

W...what?

**Comment by EzekyalLegit** [comment]
> > > > Be real though goblins will not be male only, busty green shortstacks are a very very common brand.
> > 
> > 
> > > 
> > 
> > 
> > > Note: skeletons are all male with functional genitals on them, the same cannot be said for goblins which is a tad confusing.
> > 
> > 
> > This change is how most players who want to be a female with a cock get their cock. Harvesting from skelelons.
> 
> W...what?



> > > > > > > Is this seriously a female only race? Disappointing.
> > > > 
> > > > 
> > > > > >
> > > > 
> > > > 
> > > > > >
> > > > 
> > > > 
> > > > > > They're meant to be an antithesis to goblins, which are an all male race.
> > > > 
> > > > 
> > > > > > Its something easily changed if its asked for enough
> > > > 
> > > > 
> > > > > 
> > > > 
> > > > 
> > > > > I don't see much of a reason to change it otherwise though.
> > > > 
> > > > 
> > > > So people who like faeries and roleplay males predominantly can have male Seelie characters.
> > > 
> > > 
> > > And yet anybody capable of playing this specific species to a high degree of effectiveness would also be able to roleplay as both sexes effectively, that and the fact that making seelies sexless probably wouldn't actually change anything to a significant degree because sexual features and mannerisms play such a little part in how fae act or at least what I assume how a fae should act.
> > > Either way it seems one of the species main gimmicks is sex = death so take that as you will into consideration.
> > 
> > 
> > I would play gender neutral they/them Seelie for sure.
> 
> That was the original intent, since they don't breed and propagate in a different a-sexual way. However, that turned out to be WAY harder to code unfortunately. Everything is changeable in the future though!

This is more just curiosity than anything. But what is the difficulty of being genderless, couldn't you just give them access to all genitals and make them use "they" pronouns but let them be "female" on a code level?

> > > > Be real though goblins will not be male only, busty green shortstacks are a very very common brand.
> > 
> > 
> > > 
> > 
> > 
> > > Note: skeletons are all male with functional genitals on them, the same cannot be said for goblins which is a tad confusing.
> > 
> > 
> > This change is how most players who want to be a female with a cock get their cock. Harvesting from skelelons.
> 
> W...what?

A little off topic but yes, the round start skeletons in the sewers are a good source of a free cock to sew onto yoruself.

> > > > > > > Is this seriously a female only race? Disappointing.
> > > > 
> > > > 
> > > > > >
> > > > 
> > > > 
> > > > > >
> > > > 
> > > > 
> > > > > > They're meant to be an antithesis to goblins, which are an all male race.
> > > > 
> > > > 
> > > > > > Its something easily changed if its asked for enough
> > > > 
> > > > 
> > > > > 
> > > > 
> > > > 
> > > > > I don't see much of a reason to change it otherwise though.
> > > > 
> > > > 
> > > > So people who like faeries and roleplay males predominantly can have male Seelie characters.
> > > 
> > > 
> > > And yet anybody capable of playing this specific species to a high degree of effectiveness would also be able to roleplay as both sexes effectively, that and the fact that making seelies sexless probably wouldn't actually change anything to a significant degree because sexual features and mannerisms play such a little part in how fae act or at least what I assume how a fae should act.
> > > Either way it seems one of the species main gimmicks is sex = death so take that as you will into consideration.
> > 
> > 
> > I would play gender neutral they/them Seelie for sure.
> 
> That was the original intent, since they don't breed and propagate in a different a-sexual way. However, that turned out to be WAY harder to code unfortunately. Everything is changeable in the future though!

Interesting that that is a road block. Are there simply a lot of checks in the code that call on gender? Like too many to patch up after adding a third neutral gender.

**Comment by EzekyalLegit** [comment]
> > > > > > > Is this seriously a female only race? Disappointing.
> > > > 
> > > > 
> > > > > >
> > > > 
> > > > 
> > > > > >
> > > > 
> > > > 
> > > > > > They're meant to be an antithesis to goblins, which are an all male race.
> > > > 
> > > > 
> > > > > > Its something easily changed if its asked for enough
> > > > 
> > > > 
> > > > > 
> > > > 
> > > > 
> > > > > I don't see much of a reason to change it otherwise though.
> > > > 
> > > > 
> > > > So people who like faeries and roleplay males predominantly can have male Seelie characters.
> > > 
> > > 
> > > And yet anybody capable of playing this specific species to a high degree of effectiveness would also be able to roleplay as both sexes effectively, that and the fact that making seelies sexless probably wouldn't actually change anything to a significant degree because sexual features and mannerisms play such a little part in how fae act or at least what I assume how a fae should act.
> > > Either way it seems one of the species main gimmicks is sex = death so take that as you will into consideration.
> > 
> > 
> > I would play gender neutral they/them Seelie for sure.
> 
> That was the original intent, since they don't breed and propagate in a different a-sexual way. However, that turned out to be WAY harder to code unfortunately. Everything is changeable in the future though!

I imagine all the code bits that call on gender would need to be looked at again and changed to be able to respect a third neutral option, or perhaps a null response. Which would be a huge amount of picking through code.

**Comment by Genevesh** [comment]
I'm sure someone already made a remark on the little oopsie with the config; Apparently the race itself is locked out but it's dedicated town role is available... naturally, being seelie-only. You think you could get it fixed to merge a right version for the next round?

**Comment by CalamaBanana** [comment]
There needs to be a little more leeway with the stress needed to cures people. Considering how few permanent good buffs there are, it feels the second I get tired everyone around me starts getting cursed. 

I can not fully confirm this one, but it did not seem to clear if the Seelie left them alone either. Only approaching them again with good mood fixed it, I believe.

**Comment by Coolgat3** [comment]
INSTANT SUGGESTIONS remove the ability to float above water that shit is abusable to hell.
Remove or MASSIVELY shorten the drug bullshit, it's literally so hindering they can actually win a 1v1 fight if they really try because they can spam bow

File: code/modules/mob/living/carbon/human/species_types/roguetown/other/seelie.dm [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+		OFFSET_SHIRT_F = list(0,0), OFFSET_ARMOR_F = list(0,0), OFFSET_UNDIES_F = list(0,0), \
+		)
+	specstats = list(
+		"strength" = -17,
+		"perception" = 4,
+		"intelligence" = 2,
+		"constitution" = -6,
+		"endurance" = -1,
+		"speed" = 7,
+		"fortune" = 5
```
[the remark this answers is not in the document]
Comment (Simply-Sydney) [reply code/modules/mob/living/carbon/human/species_types/roguetown/other/seelie.dm:53]: > Fortune affects STALUC which affects goodluck, at two goodluck, which Seelies are basically guaranteed to get with these fortune stats and a slight bit of mood management, they effectively have +10 damage for the purpose of considering wounds and crit chance. Essentially, the wounds they deal are more severe even with a non crit and they have an added +10% chance to get a critical hit.
> 
> It has been decided in the past that no species should be given fortune. There were heated debates about certain species having 2 fortune, let alone five.
> 
> Also their speed is flat out way too high. There is no way a person can catch them even with superior endurance and via sprinting. Seelies essentially have free reign to choose how when where and if they engage in a fight, they could grab a crossbow or bow and just go to town shooting out people's eyes with no recourse for their actions unless they accidentally run into a person or a wall.
> 
> https://github.com/Rotwood-Vale/Ratwood-Keep/blob/60ba0ea962c3db701a68094eda1416ea1e00bd43/code/modules/mob/living/living_wounds.dm#L107

Im fine lowering luck. But speed is a sticking point for me, they are extremely easy to kill and also I worry that this is going to follow the same exact trend every other race did of eventually becoming "Humen+"

**Comment by HissyNays** [comment]
I feel like they shouldn't be able to use crossbows, and likely bows too, It just makes no sense and feels too powerful. 
The luck stat is a huge increase to damage, fights are often about just whoever gets a crit first.
I don't understand why you can fuck pixies either. I feel like it should be possible with small pintle only if not possible at all.

**Comment by HissyNays** [comment]
I'm okay with fetish thigns if they make sense. But how tall are pixies supposed to be?

> > I feel like they shouldn't be able to use crossbows, and likely bows too, It just makes no sense and feels too powerful.
> > The luck stat is a huge increase to damage, fights are often about just whoever gets a crit first.
> > I don't understand why you can fuck pixies either. I feel like it should be possible with small pintle only if not possible at all.
> 
> Agreed with the first bit, the speed of a pixie favours hit and run with zero risk of retaliation.
> 
> As for the second query it's a fetish thing, people have been asking for a mechanic regarding kobolds but for understandable reasons was never even brought up here. Still it fits fluff wise with the "grim dark" setting and how terrible things get in this shitty swamp kingdom.

**Comment by pilotmans** [comment]
> I'm okay with fetish thigns if they make sense. But how tall are pixies supposed to be?
> 
> 
> 
> > > I feel like they shouldn't be able to use crossbows, and likely bows too, It just makes no sense and feels too powerful.
> 
> > > The luck stat is a huge increase to damage, fights are often about just whoever gets a crit first.
> 
> > > I don't understand why you can fuck pixies either. I feel like it should be possible with small pintle only if not possible at all.
> 
> > 
> 
> > Agreed with the first bit, the speed of a pixie favours hit and run with zero risk of retaliation.
> 
> > 
> 
> > As for the second query it's a fetish thing, people have been asking for a mechanic regarding kobolds but for understandable reasons was never even brought up here. Still it fits fluff wise with the "grim dark" setting and how terrible things get in this shitty swamp kingdom.
> 
> 
> 
> Fetish things are okay, sort of... but how tall are they? Because they look like no dick would fit in them.
> 
> 
> 
> 

Not sure but it isn't as if the pixie isn't being severely crushed from the inside out when this happens which makes me assume their about "average male pleasure toy size", whatever that may mean.

However from a gameplay standpoint this specific species seems to take on the utility role of ss13 drones but with much more flavor regarding to roleplay. And I think this is a an incredibly odd but nice flavoring to have to contrast with so many races that are just improved human.

**Comment by K4rlox** [comment]
Heres a list of issues i have with them
1. they cannot control who they curse, which can get them randomly round removed
2. they can randomly drug people, which is extremely annoying and really shitty, one of the things that gets them valided
3. the possession thing isnt well tought of, also very annoying, shouldnt be a thing

they overall seem to be a constant annoyance, as a result they are constantly valided

**Comment by Genevesh** [comment]
> Heres a list of issues i have with them
> 
> 1. they cannot control who they curse, which can get them randomly round removed
> 2. they can randomly drug people, which is extremely annoying and really shitty, one of the things that gets them valided
> 3. the possession thing isnt well tought of, also very annoying, shouldnt be a thing
> 
> they overall seem to be a constant annoyance, as a result they are constantly valided

That's kinda their whole gimmick. They're an annoying pest unless they really go out of their way to be good. You need to really be a friend to a pixie in order to reap their benefits

**Comment by Rayforx** [comment]
Pixies should not be allowed to wear any armor or have weapons larger than a knife, it's a bit silly to see pixies going into your windows with scale mail and a crossbow/spear and dodging you into the window as you play peek-a-boo the crossbow guy

**Comment by Jackofashepard** [comment]
Hey PR maker. Loving the update so far. I noticed seelies cant cast some spells while inside a lamptern like the remove clothes one. They also dont apply the blessed/cursed effect. As for the wing price i think its fine to keep it high. The price for a bandit head is 80, this isnt even an antag this is just a normal role. Chances are super high if there's even one person that liked this seelie they will make it their life mission to hunt you down, as a result i suggest a price of at least 120 mammons.

**Comment by Simply-Sydney** [comment]
> Pixies should not be allowed to wear any armor or have weapons larger than a knife, it's a bit silly to see pixies going into your windows with scale mail and a crossbow/spear and dodging you into the window as you play peek-a-boo the crossbow guy
> 
> I have to point out how LARGE of a balance concern this will be for combat when the pixie is nearly impossible to hit in terms of sprite and can range attack/go into any window while floating above tiles.

With 1str and 30% bow damage, no Seelie is doing this unless as a result of bugs and unintented exploits. People need to stop making shit up.

**Comment by Simply-Sydney** [comment]
image

Crashing this PR... with NO survivors!! (It'll be back after some rework)

**Comment by ikrarkjr** [comment]
Bring it back with male seelies next time

**Comment by Genevesh** [comment]
WOE IS ME..................

I suppose I did catch some nice rounds as a little prankster gremlin, honestly never had an issue with seelies myself, but I guess you can't stop shitters from ruining everything.

Good luck on the next seelie iteration

**Pull request closed without merging**